pkg/volume/glusterfs/glusterfs.go

```diff
@@ -383,19 +383,24 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 
 	}
 
-	//Add backup-volfile-servers and auto_unmount options.
-	options = append(options, "backup-volfile-servers="+dstrings.Join(addrlist[:], ":"))
-	options = append(options, "auto_unmount")
+	if (len(addrlist) > 0) && (addrlist[0] != "") {
+		ip := addrlist[rand.Intn(len(addrlist))]
```

Comment:
This part is already in. Please rebase to master and update this PR.

Reply:
OK, i will update it.

Reply:
Move lines of `options=...` and comments below `if (len(addrlist) > 0) && (addrlist[0] != "")` for two reasons: `addr.Delete(ip)` uses `ip` and "options" is only used in this condition.
I think "This part" here "minus/plus" is just git's difference.
Maybe i do not get your idea.